src/libcore/iter/sources.rs

```diff
@@ -377,6 +377,119 @@ pub fn once<T>(value: T) -> Once<T> {
     Once { inner: Some(value).into_iter() }
 }
 
+/// An iterator that repeats elements of type `A` endlessly by
+/// applying the provided closure `F: FnMut() -> A`.
+///
```

Comment:
It seems like his comment was copy pasted from `RepeatWith`.
It talks about endless repeating and `FnMut()`.

Reply:
@kumbayo could you file a PR to fix this?